api/pkg/s3/liststorageclasses.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s3
+
+import (
+	"net/http"
+	"encoding/xml"
+
+	"github.com/emicklei/go-restful"
+	"github.com/micro/go-log"
+	"golang.org/x/net/context"
+
+	"github.com/opensds/multi-cloud/api/pkg/policy"
+	"github.com/opensds/multi-cloud/s3/proto"
+	"github.com/opensds/multi-cloud/s3/pkg/model"
+)
+
+func (s *APIService) GetStorageClasses(request *restful.Request, response *restful.Response) {
+	if !policy.Authorize(request, response, "storageclass:get") {
```

Comment:
1. is it a standard s3 api? 
2. please add api impact in design document and swagger file

Reply:
No it's not. 
The swagger.yaml  has been updated.
Other bugs, such as missing new line at the end of file, uppercase in error messages, are fixed too.